int16 led_index
char rgb
bool enable